src/cli/flags/options.go

```diff
@@ -110,6 +112,18 @@ func AddDisableIncrementalsFlag(cmd *cobra.Command) {
 	cobra.CheckErr(fs.MarkHidden(DisableIncrementalsFN))
 }
 
+// Adds the hidden '--disable-caching' cli flag which, when set, disables
+// kopia-assisted incremental backups.
+func AddDisableAssistCachingFlag(cmd *cobra.Command) {
```

Comment:
happy to take naming suggestions if folks have them

Reply:
`Greg`, preferably.